icssc-projects/peterportal-client
# Added favicons for other platforms

# Added favicons for other platforms

## Description
Created favicons for other platforms using https://realfavicongenerator.net/

## Screenshots

No change (or noticeable change) between new and old desktop favicon
![image](https://user-images.githubusercontent.com/41417417/152887452-6f915d9d-c8bd-4e52-b6b4-c9e88f2d6146.png)
Left is new and right is old


## Steps to verify/test this change:
- Use https://realfavicongenerator.net/favicon_checker to verify that the favicon works on all major platforms


## Notes
- Previously the `href` to the favicon was prepended with `%PUBLIC_URL%/`, I am not sure if this is required since it seems to work locally on my machine.
